src/ember-app.js

```diff
@@ -54,23 +54,30 @@ class EmberApp {
    *
    * @param {string} distPath path to the built Ember app to load
    * @param {Sandbox} [sandboxClass=VMSandbox] sandbox class to use
+   * @param {Object} [sandboxGlobals={}] any additional variables to expose in the sandbox
    */
-  buildSandbox(distPath, sandboxClass) {
+  buildSandbox(distPath, sandboxClass, sandboxGlobals) {
```

Comment:
Small suggestion here: can we name the argument `additionalSandboxGlobals`?  It makes it clearer to me that these augment the existing set of globals (not replace them).